_docs/install/schedulers/openshift.md

```diff
@@ -0,0 +1,594 @@
+---
+layout: guide
+title: StorageOS Docs - OpenShift
+anchor: install
+module: install/schedulers/openshift
+---
+
+# OpenShift
+
+- [Overview](#overview)
+- [Prerequisites](#prerequisites)
+- [Examples](#examples)
+  - [Pre-provisioned Volumes](#pre-provisioned)
+    - [Pod](#pod)
+    - [Persistent Volumes](#persistent-volumes)
+  - [Dynamic Provisioning](#dynamic-provisioning)
+    - [Storage Class](#storage-class)
+- [API Configuration](#api-configuration)
+- [Examples cleanup](#examples-cleanup)
+
+## Overview
+
+StorageOS can be used as a storage provider for your OpenShift cluster.
+StorageOS runs as a container within your OpenShift environment, making local
+storage accessible from any node within the OpenShift cluster.  Data can be
+replicated to protect against node failure.
+
+At its core, StorageOS provides block storage.  You may choose the filesystem
+type to install to make devices usable from within containers.
+
+## Prerequisites
+
+OpenShift 3.7+ is required.
+
+Run the StorageOS container directly in Docker on each node,
+following the instructions at [Docker Application Container]({% link _docs/install/docker/container.md %}).
+
+>Note: OpenShift uses iptables, make sure there are rules in place to allow storageos to work.
+
+>**Note**: It is not currently possible to run the StorageOS container via
+OpenShift in a Pod or Daemonset.  StorageOS and other containerized storage
+providers require that mount propagation be enabled using the `rshared` mount
+flag.  The [containerized mount feature](https://github.com/kubernetes/community/pull/589)
+is planned for Kubernetes 1.8, and is being developed in
+[PR #46444](https://github.com/kubernetes/kubernetes/pull/46444).
```

Comment:
"Running the StorageOS container in a pod or daemonset requires Kubernetes 1.8+".

(This PR is merged and 1.8 is out now.)